packages/openapi-typescript/bin/cli.js

```diff
@@ -114,13 +114,13 @@ async function generateSchema(pathToSpec) {
     let outputFilePath = new URL(flags.output, CWD); // note: may be directory
     const isDir = fs.existsSync(outputFilePath) && fs.lstatSync(outputFilePath).isDirectory();
     if (isDir) {
-      if (typeof flags.output === 'string' && !flags.output.endsWith('/')) {
-        outputFilePath = new URL(`${flags.output}/`, CWD)
+      if (typeof flags.output === "string" && !flags.output.endsWith("/")) {
```

Comment:
It looks like we’re accidentally skipping this from linting/Prettier. Thanks for fixing this file.